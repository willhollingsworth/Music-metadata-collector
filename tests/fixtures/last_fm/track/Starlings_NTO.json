{
  "track": {
    "name": "Starlings",
    "url": "https://www.last.fm/music/NTO/_/Starlings",
    "duration": "342000",
    "streamable": {
      "#text": "0",
      "fulltrack": "0"
    },
    "listeners": "3783",
    "playcount": "11177",
    "artist": {
      "name": "NTO",
      "url": "https://www.last.fm/music/NTO"
    },
    "album": {
      "artist": "NTO",
      "title": "Starlings - Single",
      "url": "https://www.last.fm/music/NTO/Starlings+-+Single",
      "image": [
        {
          "#text": "",
          "size": "small"
        },
        {
          "#text": "",
          "size": "medium"
        },
        {
          "#text": "",
          "size": "large"
        },
        {
          "#text": "",
          "size": "extralarge"
        }
      ]
    },
    "toptags": {
      "tag": []
    }
  }
}