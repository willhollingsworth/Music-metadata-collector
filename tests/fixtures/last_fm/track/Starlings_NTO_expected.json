{
  "track_name": "Starlings",
  "url": "https://www.last.fm/music/NTO/_/Starlings",
  "artist_name": "NTO",
  "artist_url": "https://www.last.fm/music/NTO",
  "album_name": "Starlings - Single",
  "album_url": "https://www.last.fm/music/NTO/Starlings+-+Single"
}